Tools/WebKitTestRunner/cocoa/WKTextExtractionTestingHelpers.mm

```diff
@@ -26,14 +26,64 @@
 #import "config.h"
 #import "WKTextExtractionTestingHelpers.h"
 
-#import "WKTextExtractionItem.h"
 #import <objc/runtime.h>
 #import <wtf/Scope.h>
 #import <wtf/text/MakeString.h>
 #import <wtf/text/TextStream.h>
 #import <wtf/text/WTFString.h>
 #import <wtf/text/cf/StringConcatenateCF.h>
 
+// FIXME: Remove these forward declarations once text extraction support is fully migrated to WebKit from WebKitSwift.
```

Comment:
How does migrating to WebKit make it easier to access these? Would the header need to be made WebKit SPI?

Reply:
the header was in WebKit as project-level originally, and this file was able to import it (I have no idea how, but it worked). so putting it back in WK would restore this to the way it was before